about_us.html

```diff
@@ -247,137 +265,110 @@ <h1>
-                            either by applying for grants or finding industry sponsors.
+                            The finance team is the backbone of the backbone of
+                            TrickFire's technical side. Their goal is to secure
+                            funding for everything that is done in the team - to
+                            build a rover can cost tens of thousands of dollars.
+                            It is the job of this team to plan years in advance
+                            to ensure the coffers are always filled, either by
+                            applying for grants or finding industry sponsors.
                         </p>
 
-                        <img src="./assets/images/unslanted_finance.png">
+                        <img src="./assets/images/unslanted_finance.png" />
                     </div>
                 </div>
 
                 <div class="small-row-padding">
                     <div class="small-stacked-row">
-                        <h1>
-                            Outreach
-                        </h1>
+                        <h1>Outreach</h1>
 
                         <p>
-                            If a robotics team makes a rover, but no one knows about it, was the rover really built? 
-                            Our outreach team puts TrickFire Robotics on the map! Being the backbone of TrickFire’s technical side includes: 
-
-                            <br/>
-                            •Recruiting new members 
-                            <br/>
-                            •Reaching out to local industry 
-                            <br/>
-                            •Handling public relations 
-                            <br/>
-                            •Keeping our social media up to date 
-
+                            If a robotics team makes a rover, but no one knows
+                            about it, was the rover really built? Our outreach
+                            team puts TrickFire Robotics on the map! Being the
+                            backbone of TrickFire’s technical side includes:
+
+                            <br />
+                            •Recruiting new members
+                            <br />
+                            •Reaching out to local industry
+                            <br />
+                            •Handling public relations
+                            <br />
+                            •Keeping our social media up to date
                         </p>
 
-                        <img src="./assets/images/unslanted_outreach_2016.jpg">
+                        <img
+                            src="./assets/images/unslanted_outreach_2016.jpg"
+                        />
                     </div>
                 </div>
             </div>
 
             <div class="space-after-rows"></div>
         </section>
+        <h1></h1>
```

Comment:
Whats the point of `<h1></h1>`? I don't see this used anywhere else like that

Reply:
removed